js/modules/k6/grpc/stream_test.go

```diff
@@ -406,3 +407,107 @@ func TestStream_UndefinedHandler(t *testing.T) {
 
 	require.ErrorContains(t, err, "handler for \"data\" event isn't a callable function")
 }
+
+// TestStream_MetricsTagsMetadata tests that the metrics tags are correctly
+// added to samples.
+func TestStream_MetricsTagsMetadata(t *testing.T) {
```

Comment:
Seems that this test case passes even without changes [in stream.go](https://github.com/grafana/k6/pull/3746/commits/75be2cadd52216ffc89b397538c2b7a84709c10b#diff-d9e7a5b109fbc0ebc6d7bfa11c793707f250f4009d7bf24e7f545113c91aa64aR81) :thinking:

Reply:
oops should be fixed now.